junit-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/junit/xml/models/TestSuitesModel.java

```diff
@@ -0,0 +1,110 @@
+package com.github.bogdanlivadariu.reporting.junit.xml.models;
+
+import java.util.List;
+import java.util.UUID;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElementWrapper;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import com.github.bogdanlivadariu.reporting.junit.helpers.Constants;
+
+@XmlRootElement(name = "testsuites")
+@XmlAccessorType(XmlAccessType.FIELD)
+public class TestSuiteModel {
+    @XmlAttribute
+    private String failures;
+
+    @XmlAttribute
+    private String time;
+
+    @XmlAttribute
+    private String errors;
+
+    @XmlAttribute
+    private String tests;
+
+    @XmlAttribute
+    private String name;
+
+    private String uniqueID;
+
+    private String overallStatus;
+
+    @XmlElement(name = "testsuite")
+    private List<TestSuiteModel> testsuite;
+
+    public void postProcess() {
+        uniqueID = UUID.randomUUID().toString();
+        /* if total time, or failures count or errors count were not filled, we should count them by iterating through all suites*/
+        Boolean updateTime = (time == null);
+        Boolean updateFailures = (failures == null);
+        Boolean updateErrors = (errors == null);
+        if (updateTime || updateFailures || updateErrors) {
+            for (TestSuiteModel suite : testsuite) {
+                if (updateTime) {
+                    time = Double.parseDouble(time ? time : "0") + Double.parseDouble(suite.getTime());
+                }
+                if (updateFailures) {
+                    if (suite->getOverallStatus() == Constants.FAILED) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://www.codacy.com/app/bogdan-livadariu/bootstraped-multi-test-results-report/file/2685562127/issues/source?bid=3293946&fileBranchId=3336457#l52)